_aggregations/metric/average.md

```diff
@@ -55,3 +68,118 @@ GET opensearch_dashboards_sample_data_ecommerce/_search
   }
 }
 ```
+
+The aggregation name (`avg_taxful_total_price`)  can be used as a key to retrieve the aggregation from the response.
+
+## Missing values
+
+You can assign a value to missing instances of the aggregated field. See [Missing Aggregations]({{site.url}}{{site.baseurl}}/aggregations/bucket/missing/).
+
+Prepare an example by inserting an index as follows:
+
+```
+DELETE /students
+POST _bulk
+{ "create": { "_index": "students", "_id": "1" } }
+{ "name": "John Doe", "gpa": 3.89, "grad_year": 2022}
+{ "create": { "_index": "students", "_id": "2" } }
+{ "name": "Jonathan Powers", "grad_year": 2025 }
+{ "create": { "_index": "students", "_id": "3" } }
+{ "name": "Jane Doe", "gpa": 3.52, "grad_year": 2024 }
+```
+{% include copy-curl.html %}
+
+### Example: Replacing a missing value
+
+Take the average, replacing the missing GPA field with 0:
+
+```json
+GET students/_search
+{
+  "size": 0,
+  "aggs": {
+    "avg_gpa": {
+      "avg": {
+        "field": "gpa",
+        "missing": 0
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+The response is as follows. Compare to the next example, where missing values are ignored.
+
+```json
+{
```

Comment:
I've seen the response truncated to just show the aggregations section. Is this something that should be done, or do we want the entire response shown in every case?

Reply:
If the response contains a lot of extraneous information, it's fine showing it truncated. If it's a short one like this, then we can show the full response. Also, you can show responses as closed `details` blocks. That way users can expand them if they're interested. In this case, a descriptive intro sentence that spells out exactly what the users should notice is great because they can either just read the sentence or open and examine the response. For formatting the `details` block, see https://github.com/opensearch-project/documentation-website/blob/main/FORMATTING_GUIDE.md#collapsible-blocks.